vlm_pypoke/__init__.py,sha256=r8eNJT4LWbbWCMdNZjFbeMvKhfbrDTq96XVTrn7puA4,60
vlm_pypoke/client.py,sha256=RfRecv6tkQCTOKbhRqzzCECywVqkcHtzwJuuqpCm-mQ,2447
vlm_pypoke/error.py,sha256=7-NJmsXo3kFYLmkOxNkJL0yJxTV7kgb3iArhg5Witdk,263
vlm_pypoke-0.1.dist-info/METADATA,sha256=rS8qTTzyIJG9SIZyPdGE7K4J4gmLuRKVTc8VQNQ8g5s,227
vlm_pypoke-0.1.dist-info/WHEEL,sha256=G16H4A3IeoQmnOrYV4ueZGKSjhipXx8zc8nu9FGlvMA,92
vlm_pypoke-0.1.dist-info/top_level.txt,sha256=LFq9MN2RgaRCgV_4EcX9EGiWARb2bwXWsdJnd2SytAg,11
vlm_pypoke-0.1.dist-info/RECORD,,
